portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -88,10 +92,19 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
     String name = request.getParameter("name-input");
     long timestamp = System.currentTimeMillis();
 
+    Document commentDocument = Document.newBuilder().setContent(comment)
```

Comment:
It might be useful to add a comment line with a link to some reference document (api) to sentiment analysis. If you or someone wants to expand/change this it would be useful to see what api is available.

Reply:
^^ I had the same thought